Underlight/cArts.cpp

```diff
@@ -3991,7 +3991,9 @@ void cArts::EndSenseDreamers(void *value)
 	const UINT SenseDreamerIDs[] = {IDS_SENSE_DREAMERS1,IDS_SENSE_DREAMERS2,IDS_SENSE_DREAMERS3};
 	memset(message, '\0', sizeof(message));
 	//_stprintf(message, _T(""));
-
+	_stprintf(message, "You sense %d total awakened souls in the City of Dreams.", psense_msg->Total());
```

Comment:
Do strings literals need to be wrapped in a `_T()` macro?

Reply:
Meh. Maybe. Doesn't matter, _T doesn't do anything in non-unicode builds.

Reply:
yeah, but then why bother using _stprintf?